contracts/src/PBHEntryPointImplV1.sol

```diff
@@ -30,7 +30,7 @@ contract PBHEntryPointImplV1 is IPBHEntryPoint, WorldIDImpl, ReentrancyGuardTran
     /// @dev The EntryPoint where Aggregated PBH Bundles will be proxied to.
     IEntryPoint public entryPoint;
 
-    /// @notice The number of PBH transactions that may be used by a single
+    /// @notice The upper bound on the number of PBH transactions that may be used by a single
     ///         World ID in a given month.
     uint8 public numPbhPerMonth;
```

Comment:
Wdyt about adjusting the name to `pbhNonceLimit` and updating the comment to be something like "The number of PBH transactions alloted to each World ID per month, 0 indexed. For example, if the `pbhNonceLimit` is 29, a user can submit 30 PBH txs"